README.md

```diff
@@ -365,8 +367,9 @@ Services to securely store your Docker images.
 * [proxy](https://github.com/factorish/proxy) - lightweight nginx based load balancer self using service discovery provided by registrator. by [@factorish](https://github.com/factorish)
 * [registrator](https://github.com/progrium/registrator) - Service registry bridge for Docker by [@progrium][progrium]
 
-## Services for running containers
+## CaaS (Containers as a Service) - Services for running/hosting containers
```

Comment:
Sorry I should explain better. The modification of the title here must also be applied in the ToC. #services-for-running-containers ===> #caas--containers-as-a-service---services-for-running-containers so something like that....